packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -300,6 +330,14 @@ export default {
 
         shouldShowErrorPage () {
             return !this.hasCheckoutLoadedSuccessfully;
+        },
+
+        shouldShowErrorDialog () {
+            return this.nonFulfillableError && this.nonFulfillableError.showInDialog;
+        },
+
+        restaurantMenuPageUrl () {
+            return `${this.restaurant.seoName}/menu`;
```

Comment:
Is this a link to the menu page? If so they have a "restaurants-" prefix before the seoname.